pages/export.vue

```diff
@@ -285,12 +327,23 @@
     .btn-container {
         z-index: 1;
         width: calc(100% - $spacer-6);
-        position: fixed;
-        left: 50%;
-        bottom: calc(0% + $spacer-6);
-        transform: translateX(-50%);
+        height: fit-content;
```

Comment:
btn-container is called twice in this style

Reply:
It is called here and in the breakpoint. So it need to be there